.github/workflows/stale.yml

```diff
@@ -20,10 +19,13 @@ jobs:
     steps:
       - uses: actions/stale@v9
         with:
-          days-before-stale: 30
+          days-before-stale: 60
           days-before-close: -1 # Don't close anything
           stale-issue-message: "This issue has been inactive for a while. Please add a comment or provide an update if it's still relevant."
           stale-pr-message: "This PR has had no recent activity. Please update it if it's still relevant."
```

Comment:
Looks like we can skip the message: from code the label is still added even if this is missing.
Are we ok with this?
```suggestion
```